src/App.tsx

```diff
@@ -53,59 +51,54 @@ function getPreparedGoods(
 
 export const App = () => {
   const [sortField, setSortField] = useState<SortType>(SortType.None);
-  const [sortedLength, setSortedLength] = useState<SortType>(SortType.None);
   const [isReversed, setIsReversed] = useState<boolean>(false);
 
   const visibleGoods: string[] = getPreparedGoods(
-    { sortField, sortedLength, isReversed },
+    { sortField, isReversed },
   );
 
+  const handleSortAlpha = () => setSortField(SortType.Alphabetical);
+  const handleSortLenght = () => setSortField(SortType.Length);
```

Comment:
it can be one handler factory for all sort filelds. and call in jsx with proper field.
```suggestion
const handleSort = (sortField) => () => setSortField(sortField);
```